config/m4/ib.m4

```diff
@@ -105,6 +105,7 @@ AS_IF([test "x$with_ib" == xyes],
                         [AC_MSG_WARN([ibverbs header files not found]); with_ib=no])
         AC_CHECK_LIB([ibverbs], [ibv_get_device_list],
             [
+            LIBS="-libverbs $LIBS"
```

Comment:
why need both here and in IBVERBS_LDFLAGS?

Reply:
IBVERBS_LDFLAGS won't get into next libmlx5 AC_CHECK_LIB

Reply:
so need to update LDFLAGS with IBVERBS_LDFLAGS before AC_CHECK_LIB, we cannot just add libs to global list, since it will affect all targets even those which should not use verbs such as libucm

Reply:
LDFLAGS doesn't affect AC_CHECK_LIB. May pass explicitly...

Reply:
yes lets pass explicitly
AC_CHECK_LIB (library, function, [action-if-found], [action-if-not-found], **[other-libraries]**)